opentelemetry-dotnet-contrib.sln

```diff
@@ -184,9 +184,9 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Extensions",
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Extensions.Tests", "test\OpenTelemetry.Extensions.Tests\OpenTelemetry.Extensions.Tests.csproj", "{2117F4E3-6612-4E4D-A757-27271EEB7783}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Extensions.Docker", "src\OpenTelemetry.Extensions.Docker\OpenTelemetry.Extensions.Docker.csproj", "{498A6808-C0DF-441F-A764-51A3BC4B8FC5}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.ResourceDetector.Container", "src\OpenTelemetry.ResourceDetector.Container\OpenTelemetry.ResourceDetector.Container.csproj", "{498A6808-C0DF-441F-A764-51A3BC4B8FC5}"
```

Comment:
@Kielek @iskiselev We discussed this name change in the SIG meeting yesterday (1/10).

While we're pretty confident this is a better name for this package, I'd like to hold on merging it for just a bit.

I'm compiling a list of packages whose names I believe should also change. I'd like to go through that exercise first before making a final decision on this package. I'd hate to change this package only to decide later we want to change it again.

Question: What do the two of you think about releasing the package once more under the old name `OpenTelemetry.Extensions.Docker` so we can get the work from #839 released independent of the name change?

Reply:
@alanwest , I can do release with #839 functionality. I'd probably will use 1.1.0-beta as a version (so it will still be considered pre-release and will not affect on shipped/unshipped API).
As soon as we will select name, we will do 1.1.0 release (without-beta suffix).